middleware/handlers.go

```diff
@@ -51,43 +51,43 @@ func createConnection() *sql.DB {
 	return db
 }
 
-// CreateUser create a user in the postgres db
-func CreateUser(w http.ResponseWriter, r *http.Request) {
+// CreateJob create a job in the postgres db
+func CreateJob(w http.ResponseWriter, r *http.Request) {
 	// set the header to content type x-www-form-urlencoded
 	// Allow all origin to handle cors issue
 	w.Header().Set("Context-Type", "application/x-www-form-urlencoded")
 	w.Header().Set("Access-Control-Allow-Origin", "*")
 	w.Header().Set("Access-Control-Allow-Methods", "POST")
 	w.Header().Set("Access-Control-Allow-Headers", "Content-Type")
 
-	// create an empty user of type models.User
-	var user models.User
+	// create an empty job of type models.Job
+	var job models.Job
 
-	// decode the json request to user
-	err := json.NewDecoder(r.Body).Decode(&user)
+	// decode the json request to job
+	err := json.NewDecoder(r.Body).Decode(&job)
 
 	if err != nil {
 		log.Fatalf("Unable to decode the request body.  %v", err)
```

Comment:
if `err != nil`,  you may wanna return one HTTP status code that matches with the error https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html
https://blog.questionable.services/article/http-handler-error-handling-revisited/